Hazelcast.Test/Hazelcast.Client.Test/VectorClockTest.cs

```diff
@@ -0,0 +1,109 @@
+﻿/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Hazelcast.Core;
+using NUnit.Framework;
+using System.Collections.Generic;
+
+namespace Hazelcast.Client.Test
+{
+    public class VectorClockTest : SingleMemberBaseTest
+    {
+        internal static VectorClock _inst;
```

Comment:
can be private non-static with Teardown method change

Reply:
Fixed